.prettierignore

```diff
@@ -16,6 +16,7 @@
 /lib/storage/data
 
 # IDE project files
+/.idea
```

Comment:
When was prettier trying to affect these files? I agree it should not, but... I've never had this issue before

Reply:
I read the top of the `.gitignore` which stated that `If you change this file, please also modify .prettierignore`. I'll remove it.